frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -490,113 +501,135 @@ const PropertyListView: React.FC = () => {
-
-              return { rowId: change.rowId, data } as any;
-            });
-            if (changedRows.length > 0) {
-              for (const change of changedRows) {
-                const apiProperty = toApiProperty(change.data as any, true);
-                const callApi = apiProperty.parcelId ? updateParcel : updateBuilding;
-                const response: any = await callApi(apiProperty.id, apiProperty);
-                nextProperties = nextProperties.map((item, index: number) => {
-                  if (index === change.rowId) {
-                    item = {
-                      ...item,
-                      ...flattenProperty(response),
-                    } as any;
-                  }
-                  return item;
-                });
-              }
 
-              toast.info('Successfully saved changes');
-              setDirtyRows([]);
-              setData(nextProperties);
+        <Table<IProperty>
+          name="propertiesTable"
+          lockPageSize={true}
+          columns={columns}
+          data={data || []}
+          loading={data === undefined}
+          filterable
+          sort={sorting}
+          pageIndex={pageIndex}
+          onRequestData={handleRequestData}
+          pageCount={pageCount}
+          onSortChange={(column: string, direction: SortDirection) => {
+            if (!!direction) {
+              setSorting({ ...sorting, [column]: direction });
+            } else {
+              const data: any = { ...sorting };
+              delete data[column];
+              setSorting(data);
             }
           }}
-        >
-          <Form>
-            <DirtyRowsTracker setDirtyRows={setDirtyRows} />
-            <Table<IProperty>
-              name="propertiesTable"
-              lockPageSize={true}
-              columns={columns}
-              data={data || []}
-              loading={data === undefined}
-              filterable
-              sort={sorting}
-              pageIndex={pageIndex}
-              onRequestData={handleRequestData}
-              pageCount={pageCount}
-              onSortChange={(column: string, direction: SortDirection) => {
-                if (!!direction) {
-                  setSorting({ ...sorting, [column]: direction });
-                } else {
-                  const data: any = { ...sorting };
-                  delete data[column];
-                  setSorting(data);
-                }
-              }}
-              canRowExpand={(val: any) => {
-                if (val.values.propertyTypeId === 0) {
-                  return true;
-                } else {
-                  return false;
-                }
-              }}
-              detailsPanel={{
-                render: val => {
-                  if (expandData[val.id]) {
-                    return <Buildings hideHeaders={true} data={expandData[val.id]} />;
+          canRowExpand={(val: any) => {
+            if (val.values.propertyTypeId === 0) {
```

Comment:
This probably needs to include subdivisions now.

Reply:
Will subdivisions have buildings on them?

Reply:
Don't we have an `enum` for `PropertyTypes`?

Reply:
@devinleighsmith can you verify if this change need to be made, not too familiar with subdivisions in the table?